src/Entity/Adherent.php

```diff
@@ -30,7 +30,14 @@
 use JMS\Serializer\Annotation as JMS;
 
 /**
- * @ApiResource
+ * @ApiResource(
+ *     collectionOperations={},
+ *     itemOperations={
+ *         "get": {
```

Comment:
do we really need to have this endpoint? for what?

Reply:
Without this config the error `No item route associated with the type \AppBundle\Entity\Adherent` occurs while posting a vote. It's impossible to create a response when the post was successful.